src/Twig/LinkFactory.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace PhpDocumentGenerator\Twig;
+
+use PhpDocumentGenerator\ReflectionNamedType as PDGReflectionNamedType;
+use PHPStan\PhpDocParser\Ast\Type\TypeNode;
+use PHPStan\PhpDocParser\Ast\Type\UnionTypeNode;
+use ReflectionClass;
+use ReflectionNamedType;
+use ReflectionType;
+use Symfony\Component\Filesystem\Path;
+
+final class LinkFactory
+{
+    /**
+     * @return [string, string[]]
+     */
+    public function getLinksFromPHPDoc(string $phpDoc): array
+    {
+        $links = [];
+
+        // convert "@see" tags to link when possible
+        if (preg_match_all('/{@see ([^}]+)}/', $phpDoc, $matches)) {
+            foreach ($matches[0] as $i => $match) {
+                $value = $matches[1][$i];
+                $phpDoc = str_replace($match, '', $phpDoc);
+
+                // HTTP link
+                if (str_starts_with($value, 'https://') || str_starts_with($value, 'http://')) {
+                    $links[] = $value;
+                    continue;
+                }
+
+                // TODO: what goes here? @see should be a link right?
+                dd('what', $value);
```

Comment:
[`@see`](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.see.pkg.html) should be used for elements documentation, [`@link`](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.link.pkg.html) for hyperlinks.

IMHO it should be improved in another PR cause it not an easy trick: those tags must be handled inline (`{@see ...}`, `{@link ...}`) and outline, with external vs internal references.